lib/util.js

```diff
@@ -37,7 +38,7 @@ function clearRequireCacheInDir(dir, extension) {
   var files = glob.sync('**/*.' + extension, options);
 
   files.map(function(file) {
-    clearRequireCache(dir + '/' + file);
+    clearRequireCache(dir + path.sep + (file.split(/\\|\//g).join(path.sep)));
```

Comment:
@Arcath :+1:  can you please fix the build errors so that we can get this merged?